plugins/inputs/ldap/README.md

```diff
@@ -17,7 +17,7 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 
 ```toml @sample.conf
 # LDAP monitoring plugin
-[[inputs.openldap]]
+[[inputs.ldap]]
```

Comment:
This is generated from the sample.conf file. You'll need to fix it in both to get tests to pass.

Thanks again!